### ISSUE TITLE
Opening a recommended document with Nextcloud Office and closing it leaves the screen blank

### ISSUE THREAD

**Issue by matzehh84** [nextcloud/recommendations#604]
#### Step 1 (Opening the document from recommendations)
![picture_01](https://user-images.githubusercontent.com/1667923/233454725-8239359d-4881-46bc-8083-a1e82a7a4a1f.png)

#### Step 2 (Closing the document)
![picture_02](https://user-images.githubusercontent.com/1667923/233454739-ef51a825-b5ce-4096-92df-560c4471cff8.png)

#### Step 3 (Screen stays blank)
![picture_03](https://user-images.githubusercontent.com/1667923/233454748-40dfc3e4-a59e-491e-bb39-16a8172efba0.png)

### Note
The problem only occurs if the document is not located in the folder currently opened in File Browser.

### Environment
Webserver Apache2 Version 2.4.56
PHP Version php8.2-fpm
Nextcloud Version 26.0.1.1
Nextcloud Office APP Version 8.0.1
Recommendations APP Version 1.5.0
Collabora Online Development Edition: Version 22.05.12-3 (Docker)

**Comment by RuudschMaHinda** [comment]
Same issue on:
Webserver nginx Version: 1.23.4
PHP Version: 8.1.18 / php8.1-fpm
Nextcloud Version: 26.0.1.1
Nextcloud Office App Version: 8.0.1
Recommendations App Version: 1.5.0
CODE Version: 22.05.13.1 (Docker)

**Comment by vitis586** [comment]
I have the same issue with the same version as @RuudschMaHinda. Browser console shows the following error.

```
Error during post message handler 
Object { msgId: "close", args: undefined, deprecated: undefined }
 TypeError: model is null
    _updateDetailsView merged-index.js:4799
    getFileModel FilesAppIntegration.js:202
    updateFileInfo FilesAppIntegration.js:586
    close FilesAppIntegration.js:94
    close Office.vue:218
    postMessageHandler Office.vue:265
    handlePostMessage postMessage.tsx:91
    handlePostMessage postMessage.tsx:85
    e postMessage.tsx:29
postMessage.tsx:97:16
    handlePostMessage postMessage.tsx:97
    handlePostMessage postMessage.tsx:85
    e postMessage.tsx:29
```

**Comment by vitis586** [comment]
I am not seeing this behaviour anymore since circa 27.0.1. I can open the file from recommendations and then close it without the blank screen.

**Comment by joshtrichards** [comment]
Duplicate of nextcloud/richdocuments#2933
Fixed in nextcloud/richdocuments#3010
